eZ/Bundle/EzPublishCoreBundle/Command/CleanupVersionsCommand.php

```diff
@@ -91,7 +100,14 @@ protected function configure()
                 InputOption::VALUE_OPTIONAL,
                 'eZ Platform username (with Role containing at least Content policies: remove, read, versionread)',
                 self::DEFAULT_REPOSITORY_USER
-            );
+            )
+            ->addOption(
+                'excluded-content-types',
+                null,
+                InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY,
+                'List of ContentType IDs of which versions should not be removed, for instance users.',
```

Comment:
From the user point of view would be great to specify Content Type identifiers here instead of IDs.

Reply:
> From the user point of view would be great to specify Content Type identifiers here instead of IDs.

That's a very good point.